styleguide/_themes/global/scss/banners.scss

```diff
@@ -36,31 +36,125 @@ $banner-text-margin-medium: 10%;
 
 .banner-headline {
   @include heading($white);
-  font-size: 3em;
-  line-height: 1em;
+  font-size: 4rem;
+  line-height: 3.75rem;
   text-align: center;
 }
 
 
-.imacBanner-container {
-  background-color: $brand-primary;
+.imacBanner {
+  background-image: url('https://752f77aa107738c25d93-f083e9a6295a3f0714fa019ffdca65c3.ssl.cf1.rackcdn.com/bananers/backgrounds/o365-overview.jpg');
+  background-size: cover;
   color: $white;
+  margin-top: 50px;
+  overflow: hidden;
 
-  &.large {
+  &.small {
+    height: 250px;
+    padding-top: 50px;
+  }
+
+  &.medium {
     height: 300px;
-    padding-top: 30px;
+    padding-top: 50px;
+  }
+
+  &.large {
+    height: 350px;
+    padding-top: 50px;
   }
 }
 
+.imacBanner-container {
+  @include make-container;
+}
+
+.imacBanner-textRow {
+  @include make-row;
+}
+
 .imacBanner-headline {
```

Comment:
We could combine `imacBanner-headlineLarge` and `imacBanner-headlineMedium` into here so that we don't have multiple classes on those elements. That way we avoid using a class that has no styling attached to it and is only used for jQuery referencing.